tests/api/runBlock.js

```diff
@@ -0,0 +1,123 @@
+const { promisify } = require('util')
+const tape = require('tape')
+const Block = require('ethereumjs-block')
+const Transaction = require('ethereumjs-tx')
+const Common = require('ethereumjs-common')
+const util = require('ethereumjs-util')
+const runBlock = require('../../lib/runBlock')
+const StateManager = require('../../lib/stateManager')
+const runTx = require('../../lib/runTx')
+const testData = require('./testdata.json')
+const { createGenesis, createAccount, setupVM } = require('./utils')
+const { setupPreConditions } = require('../util')
+
+function setup (vm = null) {
+  // Create a mock, if no real VM object provided.
+  if (vm === null) {
+    const stateManager = new StateManager()
+    vm = {
+      stateManager,
+      emit: (e, val, cb) => cb(),
+      populateCache: stateManager.warmCache.bind(stateManager),
+      runTx: (opts, cb) => cb(new Error('test')),
+      runCall: (opts, cb) => cb(new Error('test')),
+      _common: new Common('mainnet', 'byzantium')
```

Comment:
Good that you instantiate explicitly with Byzantium here, so that we know the test data used is created in a Byzantium HF context.